extensions/istio/tests/src/test/java/io/fabric8/istio/test/v1alpha3/DestinationRuleTest.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.istio.test.v1alpha3;
+
+import static org.junit.Assert.assertTrue;
```

Comment:
We're using Junit5, let's keep all assertions consistent and use Junit5 assertions everywhere
```suggestion
import static org.junit.jupiter.api.Assertions.assertTrue;
```

Reply:
ops, thanks for spotting!